lib/tasks/populate_organisation.rake

```diff
@@ -0,0 +1,44 @@
+require 'sidekiq/api'
+#use 'rake populate_organisation:organisation' to create or update organisation details
+namespace :populate_organisation do
+  task organisation:  :environment do
+    organisation_fields_mapping = {
+      "name_en" => "name_en",
+      "name_zh_tw" => "name_zh",
+      "website" => "url",
+    }
+    #set organisation_type
+    organisation_type_name = "NGO"
+    organisation_type = OrganisationType.find_by_name_en(organisation_type_name) || OrganisationType.create(name_en: organisation_type_name ,name_zh_tw: organisation_type_name ,category_en: organisation_type_name  , category_zh_tw: organisation_type_name)
```

Comment:
When creating OrganisationType, you can leave category_en/zh_tw and name_zh blank. There is no validation for them.